returnn/tf/layers/signal_processing.py

```diff
@@ -250,18 +303,20 @@ def get_out_data_from_opts(cls, out_type={}, n_out=None, **kwargs):
 
 class MelFilterbankLayer(_ConcatInputLayer):
   """
-  This layer applies the log Mel filterbank to the input
+  This layer applies the Mel filterbank to the input.
   """
 
   layer_class = "mel_filterbank"
 
   def __init__(self, sampling_rate=16000, fft_size=1024, nr_of_filters=80, **kwargs):
     """
-    :param sampling_rate int: sampling rate of the signal which the input originates from
-    :param fft_size int: fft_size with which the time signal was transformed into the intput
-    :param nr_of_filters int: number of output filter bins
+    :param int sampling_rate: sampling rate of the signal which the input originates from
+    :param int fft_size: fft_size with which the time signal was transformed into the intput
+    :param int nr_of_filters: number of output filter bins
     """
-    def tfMelFilterBank(fMin, fMax, samplingRate, fftSize, nrOfFilters):
+
+    # noinspection PyPep8Naming,PyShadowingNames
```

Comment:
Why `PyPep8Naming`? You should never use that (unless there are very good reasons why you really need it).